README.md

```diff
@@ -1,5 +1,10 @@
 # vscode-pawn
 
+## Update 03 April 2021: 
```

Comment:
there's no precedence for a changelog in the readme, git serves as the changelog so this is unnecessary